oe_contact_forms.services.yml

```diff
@@ -7,3 +7,8 @@ services:
     class: Drupal\oe_contact_forms\Access\ContactFormsAccessCheck
     tags:
       - { name: access_check, applies_to: _oe_contact_forms_access_check }
+  oe_contact_forms.contact.commands:
```

Comment:
The name of the service can be improved. At least it should reflect the purpose of service.